var/spack/repos/builtin/packages/bison/package.py

```diff
@@ -53,6 +53,7 @@ class Bison(AutotoolsPackage, GNUMirrorPackage):
 
     provides("yacc")
 
+    depends_on("gettext")
```

Comment:
Do you know if this dependency started at some point, or if it has been there since the oldest version we support?

Reply:
Looks like it was added in 3.4 upon grepping the bison repository.